src/components/ContentWrapper.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import { DOCTYPE_CONTACTS } from '../helpers/doctypes'
+import ContactsListDataLoader from './ContactsList/ContactsListDataLoader'
+import Header from './Header'
+import Toolbar from './Toolbar'
+import { Content } from 'cozy-ui/transpiled/react/Layout'
+import { Query, useClient } from 'cozy-client'
+
+const ContentWrapper = () => {
+  const client = useClient()
+  const query = client =>
+    client
+      .all(DOCTYPE_CONTACTS)
+      .include(['accounts'])
+      .where({
+        trashed: {
+          $exists: false
+        },
+        _id: {
+          $gt: null
+        }
+      })
+      .indexFields(['_id'])
+
+  return (
+    <Query query={query(client)}>
```

Comment:
You can simply <Query query={query}> and `client` will be automatically injected. No need to get it through useClient for instance